server/index.js

```diff
@@ -70,10 +70,9 @@ const startServer = () => {
 
     if (config.server.env === 'PROD') {
       cron.schedule('*/13 * * * *', () => {
-        console.log('Pinging self...')
         axios
           .get(`${config.server.clientURL}/ping`)
-          .then(() => console.log('Self-ping successful'))
+          .then(() => {})
```

Comment:
Why?

Reply:
i figured it's not useful to keep logging the same line. It is enough to know if it fails to ping (otherwise we can assume it is fine). Also, it won't push older logs upward.